frontend/app/src/main/java/dk/scheduling/schedulingfrontend/components/DateRangePicker.kt

```diff
@@ -0,0 +1,156 @@
+package dk.scheduling.schedulingfrontend.components
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.material3.BasicAlertDialog
+import androidx.compose.material3.DateRangePicker
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.SelectableDates
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.material3.rememberDateRangePickerState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.window.DialogProperties
+import dk.scheduling.schedulingfrontend.ui.theme.SchedulingFrontendTheme
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun StandardDateRangePicker(
+    closeDialog: () -> Unit,
+    passingDate: (DateRange) -> Unit,
+    openDialog: Boolean,
+) {
+    if (openDialog) {
+        BasicAlertDialog(
+            onDismissRequest = {
+                // Dismiss the dialog when the user clicks outside the dialog or on the back
+                // button. If you want to disable that functionality, simply use an empty
+                // onDismissRequest.
+                closeDialog()
+            },
+            properties =
+                DialogProperties(
+                    usePlatformDefaultWidth = false,
```

Comment:
Hvorfor ikke default platform width? Er der for lidt plads i den?

Reply:
Nej, så stretcher den til at fylde hele skærmen men på en scuffed måde